openapi.yaml

```diff
@@ -5617,22 +5617,19 @@ components:
       description: |-
         This parameter enables pagination for the endpoint and specifies the maximum number of
         elements that arrays in the top-level object (e.g. jobs or log entries) are allowed to contain.
-        The only exception is the `links` array, which MUST NOT be paginated as otherwise the
```

Comment:
Why has this been removed? It's a crucial information.

Reply:
Ok I'll try to rewrite it a bit